apps/frontend/src/components/ui/create-project-version/CreateProjectVersionModal.vue

```diff
@@ -0,0 +1,79 @@
+<template>
+	<MultiStageModal ref="modal" :stages="stages" />
+</template>
+
+<script setup lang="ts">
+import { LeftArrowIcon, RightArrowIcon } from '@modrinth/assets'
+import { commonMessages } from '@modrinth/ui'
+
+import MultiStageModal from './MultiStageModal.vue'
+import AddChangelogStage from './stages/AddChangelogStage.vue'
+import AddDependenciesStage from './stages/AddDependenciesStage.vue'
+import AddDetailsStage from './stages/AddDetailsStage.vue'
+import AddFilesStage from './stages/AddFilesStage.vue'
+import AddMcVersionsStage from './stages/AddMcVersionsStage.vue'
+
+const { formatMessage } = useVIntl()
+
+const modal = useTemplateRef<InstanceType<typeof MultiStageModal>>('modal')
+
+const defaultNextButton = {
+	icon: RightArrowIcon,
+	label: formatMessage(commonMessages.nextButton),
+	disabled: false,
+	color: 'standard' as const,
+	iconPosition: 'after' as const,
+	onClick: () => modal.value?.nextStage(),
+}
+
+const defaultBackButton = {
```

Comment:
These could be put into the modal itself, e.g if `true` is passed rather than this btn config, it uses the defaults for that button slot